### ISSUE TITLE
Update UI to reflect DMP/HRMP message delivery fees

### ISSUE THREAD

**Issue by KiChjang** [polkadot-js/apps#9298]
After paritytech/polkadot#6843 and paritytech/polkadot#7005 have been merged, we will need to look for a place to inform the user of the delivery fees to collect for the XCM executor to send an outgoing XCM.

This new delivery fee is not included in the weight of the extrinsic and is collected separately. The formula for calculating the delivery fee is:

```
delivery_fee_factor * (base_fee + encoded_msg_len * per_byte_fee)
```

The `base_fee` is a set constant in every chain, and is defined as `CENTS * 3`. The `encoded_msg_len` can be determined simply by encoding the XCM and checking the resulting encoded blob size in bytes, and the `per_byte_fee` is just the `TransactionByteFee` which is defined as `10 * MILLICENTS` on every chain.

The only weird one is `delivery_fee_factor`, but that is also easily fetched by checking the chain state on the DMP pallet (the XCMP queue pallet if on HRMP) via the `DeliveryFeeFactor` storage map. This map is keyed on the destination's `ParaId`, and if no entries exist yet for the given parachain, then the delivery fee factor would just be `1`.

**Comment by jacogr** [comment]
Umpf, this is slightly problematic since it means for each chain we will need to know what `CENTS` refers to.

Is there no way to expose this as a runtime API, meaning that if any changes is made on the chain we don't maintain this in different places? This is really what is done with fee calculations, instead of hardcoding there is an API which means that the whole world doesn't need to keep track of how calculations are done.

**Comment by xlc** [comment]
I also wrote this issue but no one cares 😢 https://github.com/paritytech/polkadot/issues/6883

**Comment by KiChjang** [comment]
I don't believe the formula is ever going to change, but the constants may, as it has happened with Kusama. I think as long as we have a runtime API to keep track of `CENTS` and `MILLICENTS`, then we're good. Do we?

**Comment by jacogr** [comment]
Never say never - trust me on this one…
